client/src/pages/private/ParticipantTable.js

```diff
@@ -767,44 +703,62 @@ export default () => {
+                  value={reducerState.fsaText}
                   disabled={isLoadingData}
-                  onDebounce={(text) => handleFsaFilter(text)}
-                  onChange={({ target }) => setFsaText(target.value)}
+                  onDebounce={(text) => dispatch({type:'updateFilter',key:'fsaFilter', value:text})}
+                  onChange={({ target }) => dispatch({type:'updateKey',key:'fsaText',  value:target.value})}
                   placeholder='Forward Sortation Area'
                 />
               </Box>
             </Grid>
             <Grid item>
               <Box pl={2}>
-                {!hideLastNameAndEmailFilter && (
-                  <DebounceTextField
-                    time={1000}
-                    variant='filled'
-                    fullWidth
-                    value={lastNameText || ''}
-                    disabled={isLoadingData}
-                    onDebounce={(text) => handleLastNameFilter(text)}
-                    onChange={({ target }) => setLastNameText(target.value)}
-                    placeholder='Last Name'
-                  />
-                )}
+                {!hideLastNameAndEmailFilter && <DebounceTextField
+                  time={1000}
+                  variant="filled"
+                  fullWidth
+                  value={reducerState.lastNameText}
+                  disabled={isLoadingData}
+                  onDebounce={(text) => dispatch({type:'updateFilter',key:'lastNameFilter', value:text})}
+                  onChange={({ target }) => dispatch({type:'updateKey',key:'lastNameText',  value:target.value})}
+                  placeholder='Last Name'
+                />}
               </Box>
             </Grid>
             <Grid item>
               <Box pl={2}>
-                {!hideLastNameAndEmailFilter && (
-                  <DebounceTextField
-                    time={1000}
-                    variant='filled'
-                    fullWidth
-                    value={emailText || ''}
-                    disabled={isLoadingData}
-                    onDebounce={(text) => handleEmailFilter(text)}
-                    onChange={({ target }) => setEmailText(target.value)}
-                    placeholder='Email'
-                  />
-                )}
+                {!hideLastNameAndEmailFilter && <DebounceTextField
+                  time={1000}
+                  variant="filled"
+                  fullWidth
+                  value={reducerState.emailText}
+                  disabled={isLoadingData}
+                  onDebounce={(text) => dispatch({type:'updateFilter',key:'emailFilter', value:text})}
+                  onChange={({ target }) => dispatch({type:'updateKey',key:'emailText',  value:target.value})}
+                  placeholder='Email'
+                />}
+              </Box>
+            </Grid>
+            <Grid item style={{ 'marginLeft': 20 }}>
+              <Typography>Site for distance calculation: </Typography>
```

Comment:
This input got duplicated at some point

Reply:
It must have been during the rebase, I've removed it.